blocks/rss-alexa-feature-block/features/rss/xml.js

```diff
@@ -0,0 +1,162 @@
+import PropTypes from 'fusion:prop-types'
+import Consumer from 'fusion:consumer'
+import get from 'lodash/get'
+import moment from 'moment'
+import getProperties from 'fusion:properties'
+import { resizerKey } from 'fusion:environment'
+import { BuildContent } from '@wpmedia/feeds-content-elements'
+import { generatePropsForFeed } from '@wpmedia/feeds-prop-types'
+import { buildResizerURL } from '@wpmedia/feeds-resizer'
+
+const jmespath = require('jmespath')
+
+const cheerio = require('cheerio')
+
+const getURLType = (url) => {
+  const imgRegex = new RegExp(/(jpe?g|png|webp)$/);
+  const audioRegex = new RegExp('/(.mp3)$/');
+  const videoRegex = new RegExp('/(.mp4|ts)$/');
+  const defaultType = 'audio/mp3';
+  const uri = url.split(".");
+  const type = uri[uri.length -1];
+  if(imgRegex.test(url)) {
+    return `image/${type}`;
+  }
+  if(audioRegex.test(url)){
+    return `audio/${type}`;
+  }
+  if(videoRegex.test(url)){
+    return `video/${type}`;
+  }
+  
+  return defaultType;
+}
+
+const rssTemplate = (
+  elements,
+  {
+    audioAvailable,
+    channelTitle,
+    channelDescription,
+    channelCopyright,
+    channelTTL,
+    channelCategory,
+    channelLogo,
+    itemTitle,
+    pubDate,
+    itemCategory,
+    includeContent,
+    resizerURL,
+    resizerWidth,
+    resizerHeight,
+    domain,
+    feedTitle,
+    feedLanguage,
+    rssBuildContent,
+  },
+) => ({
+  rss: {
+    '@version': '2.0',
+    channel: {
+      ...(channelTitle && { title: channelTitle }),
+      link: `${domain}`,
+      ...(channelDescription && { description: channelDescription }),
+      lastBuildDate: moment
+        .utc(new Date())
+        .format('ddd, DD MMM YYYY HH:mm:ss ZZ'),
+      ...(feedLanguage && { language: feedLanguage }),
+      ...(channelCategory && { category: channelCategory }),
+      ...(channelCopyright && {
+        copyright: channelCopyright,
+      }), // TODO Add default logic
+      ...(channelTTL && { ttl: channelTTL }),
+
+      ...(channelLogo && {
+        image: {
+          url: buildResizerURL(channelLogo, resizerKey, resizerURL),
+          title: channelTitle || feedTitle,
+          link: domain,
+        },
+      }),
+
+      item: elements.map((s) => {
+        let body, category,
+        enclosureurl = jmespath.search(s, audioAvailable);
+        const url = `${domain}${s.website_url || s.canonical_url || ''}`
+        return {
+          title: jmespath.search(s, itemTitle) || '',
+          link: url,
+          guid: s._id,
+          pubDate: s[pubDate],
+          ...(enclosureurl  &&
+          {
+            enclosure: {
+              '@url': enclosureurl,
+              '@type': getURLType(enclosureurl)
+            }
+          }),
+          ...(itemCategory &&
+            (category = jmespath.search(s, itemCategory)) &&
+            category && { category: category }),
+          ...(includeContent !== 0 &&
```

Comment:
RSS feeds have to include a description.  So don't use the optional `...(includeContent !== 0`. It needs to be
`description: cheerio.load(`